packages/individual-packages/gen.js

```diff
@@ -31,6 +31,7 @@ function copyFilesRemovingDependencies() {
     const project = new lerna_project.Project('..'); // there is a lerna.json in this directory
     const packages = project.getPackagesSync();
     const unstablePackages = getUnstablePackages(packages);
+    const jsiiRosettaVersion = project.manifest.devDependencies['jsii-rosetta'];
```

Comment:
(commenting here since gh doesn't let me comment on unchanged lines)

```js
if (phase === 'phase1') {
    copyFilesRemovingDependencies();
} else {
    bringBackDependencies();
}
```

This is awkward. Split these into two separate files.

Reply:
Done.

Reply:
Done.